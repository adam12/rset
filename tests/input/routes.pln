
execute_with=doas

t460s: t460s/ common/
	common/openbsd.pln
	t460s.pln

# vim:noexpandtab:syntax=sh:ts=4
