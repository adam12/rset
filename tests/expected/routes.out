[0] t460s
common/openbsd.pln
t460s.pln
.
